tests/state/test_state_api.py

```diff
@@ -76,3 +79,24 @@ def test_request_cancel_without_backend(self, stub_broker, api_client):
         # if there is not cancel backend should raise an error
         assert res.json["error"] == "The default broker doesn't have a cancel backend."
         assert res.status_code == 500
+
+    def test_scheduled_jobs(self, stub_broker, api_client, do_work):
+        timezone = pytz.timezone("Europe/Paris")
+        scheduler = Scheduler(
+            stub_broker,
+            [
+                ScheduledJob(
+                    actor_name=do_work.actor_name,
+                    daily_time=(datetime.datetime.now(timezone) + datetime.timedelta(minutes=5)).time(),
+                    tz="Europe/Paris",
+                )
+            ],
+            period=0.1,
+        )
+        # check_redis(scheduler.client)
```

Comment:
check redis is important as it do a flushall (remove all what is stored in redis), to be sure tests don't interfere